scripts/reset-dockstoretestuser4-resources.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+STACK=$1
+
+if [ "$STACK" == "qa" ]
+then
+  URL="https://qa.dockstore.org"
+  TOKEN=${CYPRESS_QA_TOKEN}
+elif [ "$STACK" == "staging" ]
+then
+  URL="https://staging.dockstore.org"
+  TOKEN=${CYPRESS_STAGING_TOKEN}
+else
+  URL="https://dockstore.org"
+  TOKEN=${CYPRESS_PROD_TOKEN}
+fi
+
+#The function getUser takes username as argument to get the user JSON object and user id.
+function getUser {
+  USER=$(curl -X 'GET' \
+  "${URL}/api/users/username/$1" \
+  -H 'accept: application/json')
+
+  USER_ID=$(echo "$USER" | jq -r ".id")
+}
+
+
+function getUserPublishedWorkflows {
+  while IFS=$'\n' read -r line
+  do
+    PUBLISHED_WORKFLOWS_IDS+=("$line")
+  done < <(curl -X 'GET' \
+    "${URL}/api/users/${USER_ID}/workflows/published" \
+    -H 'accept: application/json' \
+    -H "Authorization: Bearer ${TOKEN}" | jq .[].id)
```

Comment:
Thanks for the hard work on this!

If you wanted to make this block a little tighter, you could simply assign the result of the command substitution to the variable, ala:
```
IDS=$(curl -X GET http://<hostname>/users/<id>/workflows/published -H 'accept: application/json' -H 'Authorization: Bearer <token>' | jq .[].id)
```
Bash drops the newlines and `IDS` is set to something like this:
```
90210 5150 777

Reply:
> Thanks for the hard work on this!

Seconding this